app/MMB/src/scenes/tripServices/__tests__/InsuranceMenuItem.test.js

```diff
@@ -0,0 +1,301 @@
+// @flow
+
+import { Alert } from '@kiwicom/mobile-localization';
+
+import { InsuranceMenuItem } from '../InsuranceMenuItem';
+
+describe('InsuranceMenuItem', () => {
+  const dateNow = Date.now;
+  let spy;
+  beforeEach(() => {
+    spy = jest.spyOn(Alert, 'translatedAlert');
+    global.Date.now = jest.fn(() => 1482363367071);
+  });
+
+  afterEach(() => {
+    spy.mockRestore();
+    global.Date.now = dateNow;
+  });
+
+  function noop() {}
+
+  it('should display if it is not a past booking, its status is confirmed, and no passengers are US citizens', async () => {
+    const dataOK = {
+      __typename: 'BookingOneWay',
+      isPastBooking: false,
+      status: 'CONFIRMED',
+      passengers: [
+        {
+          nationality: 'cz',
+        },
+        {
+          nationality: 'cz',
+        },
+      ],
+      trip: {
+        departure: {
+          localTime: '2016-12-24T23:36:07.071Z',
+        },
+      },
+    };
+    const dataPastBooking = {
+      __typename: 'BookingOneWay',
+      isPastBooking: true,
+      status: 'CONFIRMED',
+      passengers: [
+        {
+          nationality: 'cz',
+        },
+        {
+          nationality: 'cz',
+        },
+      ],
+      trip: {
+        departure: {
+          localTime: '2016-12-24T23:36:07.071Z',
+        },
+      },
+    };
+    const dataStatusPending = {
+      __typename: 'BookingOneWay',
+      isPastBooking: false,
+      status: 'PENDING',
+      passengers: [
+        {
+          nationality: 'cz',
+        },
+        {
+          nationality: 'cz',
+        },
+      ],
+      trip: {
+        departure: {
+          localTime: '2016-12-24T23:36:07.071Z',
+        },
+      },
+    };
+
+    const dataOneUSCitizen = {
+      __typename: 'BookingOneWay',
+      isPastBooking: false,
+      status: 'PENDING',
+      passengers: [
+        {
+          nationality: 'US',
+        },
+        {
+          nationality: 'cz',
+        },
+      ],
+      trip: {
+        departure: {
+          localTime: '2016-12-24T23:36:07.071Z',
+        },
+      },
+    };
+
+    // $FlowExpectedError: $refType missing
```

Comment:
could you tag it with `// $FlowRelayIssue: https://github.com/facebook/relay/issues/2394` instead?